packages/python-packages/doc-warden/warden/cmd_entry.py

```diff
@@ -40,26 +41,59 @@ def index(config):
         for pkg in packages:
             print(pkg.package_id)
 
-# verify the content of the readmes only
+# verify the content of readmes or changelogs
 def verify_content(config):
-    content_results, ignored_content_results = verify_readme_content(config)
-    output_content_results(content_results, config)
+    packages = index_packages(config)
+    if 'readme' in config.target_files[0]:
```

Comment:
I'm waffling on this one. We really only support one or another option due to the fact that we do _specific_ verification on whichever type we decide.

I'm a fan of leaving the `find_missing_files` generic, but making the argument an ENUM with only a couple options instead of an array that we grep through. It's weird to me to have specificity on how we handle them, but treat them as generic right up to that point.